helm-charts/common/whisper/values.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for whisper.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+#ASR_MODEL_PATH: ""
+
+image:
+  repository: opea/whisper:latest
```

Comment:
we should have a gaudi values for whisper, https://github.com/opea-project/GenAIComps/blob/main/comps/asr/whisper/Dockerfile_hpu

Reply:
True, but see https://github.com/opea-project/GenAIExamples/tree/main/AudioQnA/docker/gaudi#2-build-asr-image
The Gaudi image is still build with the same name as Xeon: opea/whisper:latest.
I've proposed to fix it, and will have gaudi values after that fix.